### ISSUE TITLE
Folder structure for Kinetics train, val and test data

### ISSUE THREAD

**Issue by cryptedp** [kenshohara/3D-ResNets-PyTorch#30]
For me it is not quite clear how to structure the video_directory with these datasets for kinetics. Should it be video_directory/{train,val,test}/jpg, so that it can train and validate at the same time ? or is there another folder structure i should adhere to ?

**Comment by kenshohara** [comment]
Structure of Kinetics is as the following.
video_directory
  class1
    video_id1
    video_id2
    ...
  class2
  ...
  test

Training and validation sets are located in the same directory.
DIrectory of test set is located in video_directory similar to directories of each class.

**Comment by cryptedp** [comment]
Is it normal that training with val and train in the same folder slows down training ? When I had only train videos in one folder, the training seemed to be doing fine; but since training and val data are in the same folder, my training has slowed down (0 accuracy after 10 epochs)

**Comment by kenshohara** [comment]
It is the same configuration as the official structure of Kinetics.
Such structure does not affect the training performance
because train_loader just load training videos based on kinetics.json.
So, I cannot understand why your experiment resulted in such thing...
